providers/edge/src/airflow/providers/edge/models/edge_worker.py

```diff
@@ -91,12 +92,14 @@ def __init__(
         queues: list[str] | None,
         first_online: datetime | None = None,
         last_update: datetime | None = None,
+        maintenance_comment: str = "",
```

Comment:
I'd make it nullable if not used
```suggestion
        maintenance_comment: str | None = None,
```